### ISSUE TITLE
First python example in the Simple Use section of overview.rst has a bug

### ISSUE THREAD

**Issue by stumpjumper** [RallyTools/RallyRestToolkitForPython#170]
In the file `doc/source/overview.rst`, in the Simple Use section, the first python example has a bug.  Line 45 
```
for release in response:
```
should be
```
for rls in response:
```

so the loop in the code is
```
for rls in response:
    rlsStart = rls.ReleaseStartDate.split('T')[0]  # just need the date part
    rlsDate  = rls.ReleaseDate.split('T')[0]       # ditto
    print "%-6.6s  %-16.16s   %s  -->  %s" % \
          (rls.Project.Name, rls.Name, rlsStart, rlsDate)
```

**Comment by klehman-rally** [comment]
This will be fixed in 1.5.0.  Thanks for the supplying the context and the fix!

**Comment by klehman-rally** [comment]
This has been fixed in the 1.5.0 release.
